WorkspaceServer/Kernel/CSharpKernel.cs

```diff
@@ -246,6 +248,30 @@ public Task<bool> IsCompleteSubmissionAsync(string code)
                     output,
                     command,
                     formattedValues));
```

Comment:
We should stop publishing this event here. The Jupyter layer should decide how to handle StandardOutputValueProduced.

Reply:
makes sense